jboss-tck-runner/pom.xml

```diff
@@ -417,22 +501,33 @@
                     <plugin>
                         <groupId>org.apache.maven.plugins</groupId>
                         <artifactId>maven-surefire-plugin</artifactId>
+                        <version>${surefire.version}</version>
                         <configuration>
                             <excludedGroups>${excluded.groups}</excludedGroups>
+                            <properties>
+                                <property>
+                                    <name>surefire.testng.verbose</name>
+                                    <value>1</value>
```

Comment:
Does this level of verbosity have any specific value for us (as opposed to default 0?)

Reply:
There is a problem with testng 7.4.0 that when you go above 1 in the verbose level and a test fails in a certain way, the verbose logging listener starts throwing NPEs and makes all output useless. I meant to add a comment here about 1 being the highest level and then was going to look at creating a testng patch to fix the issue. Either that, or the TCK needs to revert to 7.3.0.